### PR TITLE
Link to security hardening guidelines as a pdf 1018

### PR DESCRIPTION
**Pull request by TzviLipschitz**

### CODE REVIEW

File: content/edge/installation-bundle/edge-security-hardening.md [hunk 1]
```diff
@@ -4,4 +4,8 @@ title: Security harden the Edge appliance
 layout: redirect
 ---
 
-For information about security configuration, see [Configuring security](/edge/configuration/#configuring-security). 
\ No newline at end of file
+{{< c8y-admon-important >}}
+For information about deployment security best practices and tenant administration security configuration guidelines, see [Security Hardening Guidelines](https://empower.softwareag.com/sl24sec/SecuredServices/document/java/cumulocity_iot_platform/iot10-15-0/10-15-0_Security_Hardening_Guidelines_guide.pdf).
```
Comment (BeateRixen) [inline content/edge/installation-bundle/edge-security-hardening.md:8]: @TzviLipschitz This documentation is not publicly available so we should not link to it.

Comment (BeateRixen) [inline content/edge/installation-bundle/edge-security-hardening.md:8]: @TzviLipschitz This documentation is not publicly available so we should not link to it.

Comment (MWindel) [inline content/edge/installation-bundle/edge-security-hardening.md:8]: @TzviLipschitz This link needs to be removed in this version as well as for the new structure (develop)

**Review by bbyreddy — APPROVED** [review]

File: content/edge/configuration-bundle/configuring-security.md [hunk 2]
```diff
@@ -8,23 +8,22 @@ The Edge appliance is security hardened by default.
 Furthermore, you can apply specific security configurations to enhance the security of your Edge appliance.
 
 {{< c8y-admon-important >}}
-For information about deployment security best practices and tenant administration security configuration guidelines, see [Security Hardening Guidelines](https://empower.softwareag.com/sl24sec/SecuredServices/document/java/cumulocity_iot_platform/iot10-15-0/10-15-0_Security_Hardening_Guidelines_guide.pdf).
+For information about deployment security best practices and tenant administration security configuration guidelines, see [Security Hardening Guidelines](/files/edge/Security_Hardening_Guidelines_guide_1018.pdf).
```
Comment (MWindel) [inline content/edge/configuration-bundle/configuring-security.md:11]: @TzviLipschitz The problem with showing PDF is a) it is internal information and b) it needs to be updated constantly whenever we upload a new PDF. I would highly suggest to remove it from the documentation and repo.

**Review by MWindel — APPROVED** [review]

**Pull request merged**